### PR TITLE
feat(repositories): Add Planets repository

### PR DESCRIPTION
**Pull request by flemzord**
## Description

Add _planets_ repository

## Breaking Changes
~

### Checklist

* [x] `terraform fmt` and `terraform validate` both work from the root directory (look in CI for an example)
* [x] Docs have been added/updated (for bug fixes/features)
* [x] Any breaking changes are noted in the breaking changes section above

### CODE REVIEW

File: repositories.tf [hunk 1]
```diff
@@ -164,3 +164,24 @@ module "wheel-of-fortune" {
 
   repository-issue_labels = concat(var.default-issue_labels)
 }
+
+
```
Comment (snroki) [inline repositories.tf:168]: one blank line is enough :)

File: teams.tf [hunk 2]
```diff
@@ -172,13 +173,16 @@ module "developers" {
...
     module.wheel-of-fortune.name,
+    module.planets.name,
   ]
 
   team-repositories_permission = {
     (module.github.name)               = "push",
     (module.blog_eleven-labs_com.name) = "push",
     (module.codelabs.name)             = "push",
     (module.wheel-of-fortune.name)     = "push",
+    (module.wheel-of-fortune.name)     = "push",
```
Comment (JeremyGreaux) [inline teams.tf:184]: Why ?

Comment (VEBERArnaud) [reply teams.tf:184]: fail

File: repositories.tf [hunk 3]
```diff
@@ -164,3 +164,38 @@ module "wheel-of-fortune" {
...
+  repository-auto_init      = false
+  repository-default_branch = "master"
+
+  # branches protection
+  branches_protection = [
+    {
+      branch                                     = "master"
+      enforce_admins                             = true
+      require_signed_commits                     = true
```
Comment (snroki) [inline repositories.tf:184]: do we really want to enforce signed commits on planets ?

File: repositories.tf [hunk 4]
```diff
@@ -164,3 +164,38 @@ module "wheel-of-fortune" {
...
+      status_check-strict                        = false
+      status_check-contexts                      = []
+      pr_reviews-required_approving_review_count = 1
+      pr_reviews-require_code_owner_reviews      = false
```
Comment (snroki) [inline repositories.tf:188]: maybe we should enforce the coder_owner reviews for this project

File: repositories.tf [hunk 5]
```diff
@@ -164,3 +164,38 @@ module "wheel-of-fortune" {
...
+  repository-auto_init      = false
+  repository-default_branch = "master"
+
+  # branches protection
+  branches_protection = [
+    {
+      branch                                     = "master"
+      enforce_admins                             = true
+      require_signed_commits                     = true
+      status_check-strict                        = false
```
Comment (snroki) [inline repositories.tf:185]: this means we can merge without the CI ?

Comment (VEBERArnaud) [reply repositories.tf:185]: https://www.terraform.io/docs/providers/github/r/branch_protection.html#strict

File: repositories.tf (hunk 3 above)
Comment (VEBERArnaud) [reply repositories.tf:184]: as you want

File: repositories.tf (hunk 4 above)
Comment (VEBERArnaud) [reply repositories.tf:188]: https://www.terraform.io/docs/providers/github/r/branch_protection.html#require_code_owner_reviews

**Pull request merged**